File: java/jimmer-sql/model/src/main/java/org/babyfish/jimmer/sql/example/model/Gender.java
package org.babyfish.jimmer.sql.example.model;

import org.babyfish.jimmer.sql.EnumItem;

public enum Gender {

    @EnumItem(name = "M")
    MALE,

    @EnumItem(name = "F")
    FEMALE
}

/*----------------Documentation Links----------------
❶ ❷ https://babyfish-ct.github.io/jimmer-doc/docs/mapping/advanced/enum
---------------------------------------------------*/

File: kotlin/jimmer-sql-kt/model/src/main/kotlin/org/babyfish/jimmer/sql/example/kt/model/Gender.kt
package org.babyfish.jimmer.sql.example.kt.model

import org.babyfish.jimmer.sql.EnumItem

enum class Gender {

    @EnumItem(name = "M") // ❶
    MALE,

    @EnumItem(name = "F") // ❷
    FEMALE
}

/*----------------Documentation Links----------------
https://babyfish-ct.github.io/jimmer-doc/docs/mapping/advanced/enum
---------------------------------------------------*/